lib/Extractor/Candidates.cpp

```diff
@@ -151,7 +156,11 @@ Inst *ExprBuilder::makeArrayRead(Value *V) {
   if (Opts.NamedArrays)
     Name = V->getName();
   unsigned Width = DL.getTypeSizeInBits(V->getType());
-  return IC.createVar(Width, Name);
+  // Compute known bits
+  APInt KnownZero(Width, 0), KnownOne(Width, 0);
```

Comment:
why are you using the two-argument constructor here and the three-argument constructor elsewhere?